gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -187,6 +188,7 @@ void GazeboRosApiPlugin::loadGazeboRosApiPlugin(std::string world_name)
   light_modify_pub_ = gazebonode_->Advertise<gazebo::msgs::Light>("~/light/modify");
   request_pub_ = gazebonode_->Advertise<gazebo::msgs::Request>("~/request");
   response_sub_ = gazebonode_->Subscribe("~/response",&GazeboRosApiPlugin::onResponse, this);
+  rtf_sub_ = gazebonode_->Subscribe("~/world_stats", &GazeboRosApiPlugin::publishRealTimeFactor, this);
```

Comment:
I believe this can be done without subscribing to world_stats (just use ```world_->GetSimTime()``` and ```world->GetRealTime()``` like in the publishSimTime function.

Reply:
:+1: